images/1.7.6.3/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM prestashop/base:5.6-apache
+LABEL maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"
```

Comment:
@Quetzacoalt91 Are you still OK to be labeled the maintainer of these images ? Maybe one day we should put a more generic email address, such as core@prestashop.com or docker-core@prestashop.com ?

Reply:
Yes we should have a generic address, probably `docker@prestashop` as several teams could manage this images. WDYT?

Reply:
See https://github.com/PrestaShop/docker/pull/202 (email address was chosen by Pablo)